main.tf

```diff
@@ -4,7 +4,7 @@ provider "aws" {
 
 resource "aws_instance" "webserver" {
   ami           = "ami-01e36b7901e884a10"
-  instance_type = "t2.micro"
+  instance_type = "t3.micro"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure virtual private cloud (VPC) is configured for AWS EC2 instances**
 
**Vulnerability**
By default AWS EC2 instances are launched in default VPC which is not recommended for production grade. AWS VPCs provides the controls to facilitate a formal process for approving and testing all network connections and changes to the firewall and router configurations.
 
**Remediation**
Migrate from EC2 classic to VPCs. 1: Create a VPC. 2: Configure Your Security Group. 3: Create an AMI from Your EC2-Classic Instance. 4: Launch an Instance Into Your VPC.

source: accurics-stageone